pkg/daemons/agent/agent_windows.go

```diff
@@ -16,8 +16,12 @@ import (
 	"k8s.io/kubernetes/pkg/kubeapiserver/authorizer/modes"
 )
 
+var (
+	networkName string = "vxlan0"
```

Comment:
Is there any way to get rid of this package level variable? If not, can we export it and set it directly and delete the setter?

Reply:
I didn't really want to go this route.  I needed to be able to override this in RKE2 for the different CNIs we will be supporting. This is a pattern that I seen before, so i will set it directly and remove the setter.